forge-gui/res/cardsfolder/f/freyalises_winds.txt

```diff
@@ -3,7 +3,7 @@ ManaCost:2 G G
 Types:Enchantment
 T:Mode$ Taps | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever a permanent becomes tapped, put a wind counter on it.
 SVar:TrigPutCounter:DB$ PutCounter | Defined$ TriggeredCardLKICopy | CounterType$ WIND | CounterNum$ 1
-R:Event$ Untap | ActiveZones$ Battlefield | ValidCard$ Permanent.counters_GE1_WIND | ReplaceWith$ RepRemoveCounter | UntapStep$ True | Description$ If a permanent with a wind counter on it would untap during its controller's untap step, remove all wind counters from it instead.
+R:Event$ Untap | ActiveZones$ Battlefield | ValidCard$ Permanent.counters_GE1_WIND | ReplaceWith$ RepRemoveCounter | UntapStep$ Player | Description$ If a permanent with a wind counter on it would untap during its controller's untap step, remove all wind counters from it instead.
```

Comment:
and perhaps instead of UntapStep just use `ActivePhases$ Untap`

Reply:
This probably would collide with `Bewitching Leechcraft` one?

The ReplacementEffect that does still untap for a cost?

Reply:
hmn good point, the result would change (though I'm not sure which one the rules might actually consider valid)

Reply:
my gut is telling me, it should be affected by it